### PR TITLE
Sequence Labelling with a BiLSTM in PyTorch

### PR DESCRIPTION
**Pull request by krishnapalS**
In this notebook we'll explore how we can use Deep Learning for sequence labelling tasks such as part-of-speech tagging or named entity recognition. We won't focus on getting state-of-the-art accuracy, but rather, implement a first neural network to get the main concepts across.

### CODE REVIEW

**Comment by krishnapalS** [comment]
@nikhilkala @cclauss Please review this PR

**Comment by cclauss** [comment]
https://github.com/TheAlgorithms/Python/issues/2510#issuecomment-701540969

**Review by nikhilkala — APPROVED** [review]
